src/index.js

```diff
@@ -118,6 +118,9 @@ function getConfigurationFromPluginOpts(pluginOpts) {
           const configOverrideSource = fs.readFileSync(resolvedPath, 'utf-8')
           const configOverride = JSON.parse(configOverrideSource)
           const mergedConfig = mergeDeep(primaryConfig, configOverride)
+          //Overwrite source config file 
+          const configPath = path.resolve(configuration.configPath)
```

Comment:
Is the config path already defined+resolved somewhere? If so let's use that value or elevate it so it can be used here. If that doesn't sound like a good idea, explain why

Reply:
I got rid of `path.resolve`. configuration.configPath is where it's defined